sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -265,6 +268,23 @@ builder.Services.AddOpenTelemetry().UseAzureMonitor(o =>
 });
 ```
 
+#### Drop a Metrics Instrument
```

Comment:
what is the reason behind documenting this in the distro readme?

Reply:
We are adding additional metrics here. If customers are not interested in collecting them, we are providing an option to drop them. This will also serve as a reference for us to include in the public documentation.

Reply:
what is meant by "additional" here? The distro is opinionated, and brings in httpclient/aspnetcore. So having spans and metrics from those libraries is **normal** behavior right? For example, there is no doc to show "how not to add spans from httpclient". So we don't need it for metrics as well.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore#list-of-metrics-produced
_"The library includes additional metrics beyond those defined in the [specification](https://github.com/open-telemetry/semantic-conventions/blob/v1.23.0/docs/http/http-metrics.md)"_

It makes sense for the instrumentation libraries to talk about view recipe, because it is doing things **beyond** the official spec/conventions.

Reply:
I strongly prefer to keep this section because, in .NET 8.0, `Microsoft.AspNetCore.Hosting` meter introduces these two instruments. If customers want to opt out of `http.server.active_requests`, I believe there should be documentation available to guide them.

http.server.request.duration
http.server.active_requests

Reply:
Please make it a specific thing then - i.e if this distro is adding "extra" things, then document how to remove that specific thing. 
General usage of View is someone else's (otel sdk) doc problem. Feel free to link to that from here, but copying that content here is not required.

Reply:
something like: 
Distro enables metrics collection and collects more metrics than otel convention. Specifically it collects "http.server.active_requests", that is not part of otel conventions. Use the following line to remove that metrics
```
view recipe to remove the particular metric.
```
Also link to otel view docs.